hal/nrf_temp.h

```diff
@@ -70,6 +70,37 @@ static __INLINE int32_t nrf_temp_read(void)
     return ((NRF_TEMP->TEMP & MASK_SIGN) != 0) ? (int32_t)(NRF_TEMP->TEMP | MASK_SIGN_EXTENSION) : (NRF_TEMP->TEMP);
 }
 
+/**
+ * @brief Function to trigger start of temperature measurement.
+ *
+ * This function triggers the Temp peripheral to start measuring temperature. The function also clears the DATARDY event register before starting the one-shot operation.
```

Comment:
Please use TEMP instead of Temp peripheral name. Additionally break the line at 100th character (or earlier)